app/controllers/users/registrations_controller.rb

```diff
@@ -1,6 +1,11 @@
-# rubocop: disable Rails/LexicallyScopedActionFilter
 class Users::RegistrationsController < Devise::RegistrationsController
-  after_action :send_welcome_email, only: [:create]
+  def create
```

Comment:
Does this tweak need a spec, or is it already covered?

Reply:
Nice catch - I incorrectly thought this was covered in the system tests. Added a request spec to cover the 3 possible 3 different branches.